src/hotkey.rs

```diff
@@ -92,20 +92,13 @@ impl HotKey {
         }
 
         let mut hotkey = Self { mods, key, id: 0 };
-        hotkey.id = hotkey.generate_hash();
+        hotkey.id = (mods.bits() as u64) << 32 | key as u64;
         hotkey
```

Comment:
On a second thought, I think we can fit the whole id into a u32, as `Code` only contains a 215 variant, so the maximum possible value is `214` which is no where near the max value of a u16 (65535), same can be said for modifiers. 
```rs
        hotkey.id = mods.bits() << 16 | key;
```

We can also simplify this code a bit to be 

```suggestion
        Self { mods, key, id: mods.bits() << 16 | key }
```